examples/controls_example/public/react_controls/control_group/init_controls_manager.ts

```diff
@@ -85,7 +88,7 @@ export function initControlsManager(initialControlPanelsState: ControlPanelsStat
   }
 
   return {
-    controlsInOrder$: controlsInOrder$ as PublishingSubject<Array<{ id: string; type: string }>>,
+    controlsInOrder$,
```

Comment:
I chose to make this a public `BehaviorSubject` rather than adding a setter for the control order - this felt cleaner to me.